app/Models/Context.php

```diff
@@ -55,7 +55,7 @@ class FreshRSS_Context {
 	/**
 	 * Initialize the context for the global system.
 	 */
-	public static function initSystem($reload = false) {
+	public static function initSystem($reload = false): ?FreshRSS_SystemConfiguration {
```

Comment:
Please check if this syntax works with PHP 7.2. I am not sure it does

Reply:
this syntax work for PHP7.2 since PHP7.1 https://kinsta.com/blog/php-7-1-0/#nullable-types

Reply:
https://www.php.net/manual/en/migration71.new-features.php